app/controllers/tutorials_controller.rb

```diff
@@ -38,9 +39,22 @@ def edit_confirmation
   end
 
   def view
+    @current_user = User.find_by_id(current_user)
     @tutorial = Tutorial.find_by_id(params[:tutorial_id])
     @chapters = @tutorial.chapters.order('id')
-    render "view"
+    @warning_count = 0
+    access_check = @tutorial.privacy_check(@current_user)
+    draft_check = @tutorial.draft_check
+
+    if access_check == false
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.